src/devices/Ili9341/samples/Program.cs

```diff
@@ -10,7 +10,7 @@
 using System.Threading.Tasks;
 using Iot.Device.Ft4222;
 using Iot.Device.Ili9341;
-
+#pragma warning disable CA1416 // Temporarily, will be removed after binding is updated
```

Comment:
Can you either explain what is this for or add a link to an issue to track us actually doing this work?

Reply:
This will be fixed with #2084, which shall be reviewed and merged just after.